packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -234,6 +234,9 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
         if (this.configurations.isSectionName(name)) {
             return true;
         }
+        if (Object.keys(this.combinedSchema.properties).some(p => p.startsWith(name + '.'))) {
```

Comment:
Could be expensive with many preferences?

Reply:
removed it. no longer necessary with `style` option in `PreferenceProxy`.